### PR TITLE
agbalumo: Version 1.000; ttfautohint (v1.8.4) added

### LINKED ISSUE #6515: Add Agbalumo
**Font Project Git Repo URL:** https://github.com/SorkinType/Agbalumo

**Super short description of the Font Family:**

This project is part of the African scripts effort currently underway.

Curvy, chunky, and compact. The Agbalumo display typeface has been designed to represent and capture the beauty of African languages. Primarily taking shape from the use of a brush pen, it's charming, playful, and cute look lends itself to a variety of commercial use cases. From beautifully curated eye catching websites to brightly colored packaged items, if you aim to communicate personality, Agbalumo is a sure selection.

Agbalumo is a single weight font. The glyph set includes standard opentype features and an assortment of stylistic alternates. Agbalumo can be used for African languages that make use of the Latin script, European languages, and Vietnamese.

Agbalumo is designed by ColumnType

**Requirements:**

I understand that Google Fonts will publish only fonts that matches [its requirements](https://googlefonts.github.io/gf-guide/), and I can confirm the project meets them (by ticking the cases, or putting x between the square brackets in text mode):

- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to namecheck.fontdata.com
- [x] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [x] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [x] I read, agree and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/onboarding.html)
- [x] I am the author of the font or the author agreed to this submission
- [x] I will maintain the repository

**Image:**

image

### PR DESCRIPTION
**Pull request by eliheuer**
8a7215a: [gftools-packager] agbalumo: Version 1.000; ttfautohint (v1.8.4) added

* agbalumo Version 1.000; ttfautohint (v1.8.4) taken from the upstream repo https://github.com/SorkinType/Agbalumo.git at commit https://github.com/SorkinType/Agbalumo/commit/3e64aac2795e538846e6412d444d307f345108ec.

Resolves #6515

### CODE REVIEW

**Comment by eliheuer** [comment]
Updated

## Agbalumo: Version 1.000; ttfautohint (v1.8.4) added

---
 0215c94: [gftools-packager] Agbalumo: Version 1.000; ttfautohint (v1.8.4) added

* Agbalumo Version 1.000; ttfautohint (v1.8.4) taken from the upstream repo https://github.com/SorkinType/Agbalumo.git at commit https://github.com/SorkinType/Agbalumo/commit/3e64aac2795e538846e6412d444d307f345108ec.

**Comment by eliheuer** [comment]
Updated

## agbalumo: Version 1.000; ttfautohint (v1.8.4) added

---
 c8dfc0c: [gftools-packager] agbalumo: Version 1.000; ttfautohint (v1.8.4) added

* agbalumo Version 1.000; ttfautohint (v1.8.4) taken from the upstream repo https://github.com/SorkinType/Agbalumo.git at commit https://github.com/SorkinType/Agbalumo/commit/ae5e87904dec7e521a9c565918e31e7b81581f2e.

**Comment by eliheuer** [comment]
Updated

## Agbalumo: Version 1.000; ttfautohint (v1.8.4) added

---
 b49e64f: [gftools-packager] Agbalumo: Version 1.000; ttfautohint (v1.8.4) added

* Agbalumo Version 1.000; ttfautohint (v1.8.4) taken from the upstream repo https://github.com/SorkinType/Agbalumo.git at commit https://github.com/SorkinType/Agbalumo/commit/ae5e87904dec7e521a9c565918e31e7b81581f2e.

**Comment by eliheuer** [comment]
This project used to be called Danfo, but the name was recently changed to Agbalumo.

Here is the add font issue: https://github.com/google/fonts/issues/6515

File: ofl/agbalumo/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<p>Curvy, chunky, and compact. The Agbalumo display typeface has been designed to represent and capture the beauty of African languages. Primarily taking shape from the use of a brush pen, it's charming, playful, and cute look lends itself to a variety of commercial use cases. From beautifully curated eye catching websites to brightly colored packaged items, if you aim to communicate personality, Agbalumo is a sure selection.</p>
+
+<p>Agbalumo is a single weight font. The glyph set includes standard opentype features and an assortment of stylistic alternates. Agbalumo can be used for African languages that make use of the Latin script, European languages, and Vietnamese.</p>
+
+<p>Agbalumo is designed by <a href="https://www.instagram.com/columntype">ColumnType</a></p>
```
Comment (RosaWagner) [inline ofl/agbalumo/DESCRIPTION.en_us.html:5]: @eliheuer the Instagram page should be part of the author's biography. This form should be fill up by the author: https://docs.google.com/forms/d/e/1FAIpQLSeMwHN8J213ZaxHrr5lHCrX56HY_NjGrWB8o604g98YxuMrdA/viewform. It is linked to a spreadsheet I shared with you. This article will help to add the bio: https://googlefonts.github.io/gf-guide/profile.html.

Having ColumnType in the description and Raphael Alegbeleye in the metadata.pb is a bit confusion. The author should decide on how they want to be credited.

File: ofl/agbalumo/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The Danfo Project Authors (https://github.com/googlefonts/googlefonts-project-template)
```
Comment (RosaWagner) [inline ofl/agbalumo/OFL.txt:1]: Missing correct URL and copyright string needs be updated with the new name.

**Comment by RosaWagner** [comment]
Since the font is quite thick, we need to decide if it should have "One" appended to the name to be able to onboard other weights later.

**Review by RosaWagner — CHANGES_REQUESTED** [review]

**Comment by RosaWagner** [comment]
If the "development" panel doesn't work to link the issue to can add the issue number with a closing word to achieve the linking :)

File: ofl/agbalumo/METADATA.pb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: "Agbalumo"
+designer: "Raphael Alegbeleye"
```
[the remark this answers is not in the document]
Comment (davelab6) [reply ofl/agbalumo/METADATA.pb:2]: @ebensorkin

**Comment by eliheuer** [comment]
> Since the font is quite thick, we need to decide if it should have "One" appended to the name to be able to onboard other weights later.

This was resolved in various chats today, we are going to go ahead with onboarding this as just "Agbalumo" without "One" appended. The docs might need to be updated to make it clear this requirement isn't absolutely necessary and exceptions can be made.

File: ofl/agbalumo/DESCRIPTION.en_us.html (hunk 1 above)
Comment (eliheuer) [reply ofl/agbalumo/DESCRIPTION.en_us.html:5]: The Instagram link was fixed in this PR: https://github.com/SorkinType/Agbalumo/pull/16

I talked with the designer about the designer profile, and he said he completely filled out the form. I'm confirming I see his info in the Designer Profile Info sheet.

File: ofl/agbalumo/OFL.txt (hunk 2 above)
Comment (eliheuer) [reply ofl/agbalumo/OFL.txt:1]: This was fixed in this upstream PR: https://github.com/SorkinType/Agbalumo/pull/17

File: ofl/agbalumo/METADATA.pb (hunk 3 above)
[the remark this answers is not in the document]
Comment (eliheuer) [reply ofl/agbalumo/METADATA.pb:2]: I talked to Dave and Eben about this, Raphael should be Principal Design, but Eban should be included as a secondary designer.

[the remark this answers is not in the document]
Comment (eliheuer) [reply ofl/agbalumo/METADATA.pb:2]: Will update this in the next PR

**Review by eliheuer — COMMENTED** [review]
I resolved all the issues brought up by Rosalie and Dave, making a new pull request soon for a second review.

**Comment by eliheuer** [comment]
@RosaWagner About the designer credits, Raphael should be Principal Design, but Eben should be included as a secondary designer.

I included both Sorkin Type and Eben Sorkin in the credits because the font is hosted on the Sorkin Type, lists Sorkin Type as the manufacturer, and other similar fonts already on Google fonts do the same thing, I looked at Calistoga as an example: https://fonts.google.com/specimen/Calistoga/about

ColumnType is a label Raphael wants to use to release more fonts under. The ColumnType website is not live, and the only online presence in an Instagram page with a few posts: https://www.instagram.com/columntype

**Comment by RosaWagner** [comment]
I see only one small issue so it won't block the merging, but maybe @eliheuer you can raise an issue upstream so it can be fixed in a next update.

There is an inconsistency between `Ldot` and `L periodcentered.loclCAT.case`; 
1. since the `locl` feature is making the substitution with `periodcentered.loclCAT`, `Ldot` is a bit useless here, it can be kept for backward compatibility or removed.
2. `L periodcentered.loclCAT.case  L` doesn't display the dot in the middle of the 2 L.

A good practice would be to achieve that through with sidebearing of `periodcentered.loclCAT.case`, then if Ldot is meant to stay, it can be a component of `L periodcentered.loclCAT.case` with automatic alignment to make sure both strings are consistent.

Screenshot 2023-10-10 at 09 20 03

**Review by RosaWagner — APPROVED** [review]

**Comment by eliheuer** [comment]
Thanks! I filled an [issue upstream](https://github.com/SorkinType/Agbalumo/issues/19). Eben and the designer are planning on a v 1.100 update this year, so I will try to make sure this is fixed.

**Pull request merged**